src/templates/kit/screens/TokensScreen.js

```diff
@@ -487,9 +487,18 @@ function formatReviewFields(screenData) {
       'Token name & symbol',
       `${screenData.tokenName} (${screenData.tokenSymbol})`,
     ],
-    ...screenData.members.map(([account], i) => [
+    ...screenData.members.map(([account, amount], i) => [
       `Tokenholder #${i + 1}`,
-      <IdentityBadge entity={account} />,
+      <div css="display: flex">
+        <IdentityBadge entity={account} />
```

Comment:
Eventually we can use custom labels here too so that people can start giving alias to addresses right away.

Reply:
Yes that would be great! But it would also require to use a temporary ID to these apps, as they don’t have an address yet at this point.

Reply:
Aren't these the addresses for the token holders? Why would we need an ID for apps?
I was thinking on being able to add custom labels to new token holders on an organization as it is being created so that the user does not have to do that later on.

Reply:
Ah yes sorry, I thought we were on the AppBadge!